lib/domodule.js

```diff
@@ -46,4 +109,47 @@ class Domodule {
   find(selector) {
     return this.el.querySelectorAll(selector);
   }
+
+  findOne(selector) {
+    const found = this.find(selector);
+
+    if (found.length) {
+      return found[0];
+    }
+
+    return null;
+  }
+
+  findByName(name) {
+    return this.els[name];
+  }
+
+  getOption(option) {
+    return this.options[option];
+  }
+
+  // static methods can't access `this` so they go last
+  static getInstance(name) {
```

Comment:
we should add support for also passing in an element node and returning the instance